keyboards/levinson/keymaps/jyh2/keymap.c

```diff
@@ -0,0 +1,241 @@
+#include "levinson.h"
```

Comment:
This file isn't necessary and won't get processed.

Reply:
@drashna You sure about that?

Reply:
Sorry, nope, misread the 2....

But this file has all of the same issues as the other, and should be updated in the same way